src/main/resources/wa-task-configuration-publiclaw-care_supervision_epo.dmn

```diff
@@ -1103,6 +1104,23 @@
           <text>true</text>
         </outputEntry>
       </rule>
+      <rule id="DecisionRule_1sumgf3">
+        <inputEntry id="UnaryTests_1oc8kos">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0qd59ye">
+          <text>"viewAdditionalApplications","reviewMessageCTSC","reviewMessageLegalAdviser","reviewMessageHearingCentreAdmin","reviewMessageHearingJudge","reviewMessageAllocatedJudge","approveOrders","reviewOrderCMO"</text>
```

Comment:
This was my bad, missed out the 'Review Response' tasks in the ticket description - but seeing all these tasks here, we should also probably add the date for Review Response too, otherwise messages/responses will look different!